src/main/java/com/dev/cinema/exception/HashGeneratingException.java

```diff
@@ -0,0 +1,8 @@
+package com.dev.cinema.exception;
+
+public class HashGeneratingException extends Exception {
```

Comment:
remove this exception please. it is redundant

Reply:
done